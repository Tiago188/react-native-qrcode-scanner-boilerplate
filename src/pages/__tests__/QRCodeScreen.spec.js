import React from 'react';
import { shallow } from 'enzyme';

import QRCodeScreen from '../QRCodeScreen';

const props = {

    

}

describe('<QRCodeScreen>', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<QRCodeScreen />);
    expect(wrapper).toMatchSnapshot();
  });

});
