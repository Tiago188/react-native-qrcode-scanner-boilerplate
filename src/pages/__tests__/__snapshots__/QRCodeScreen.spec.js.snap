// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QRCodeScreen> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "black",
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <QRCodeScanner
    bottomContent={
      <View
        style={
          Object {
            "padding": 16,
          }
        }
      />
    }
    cameraProps={Object {}}
    cameraStyle={
      Object {
        "height": 1334,
      }
    }
    cameraType="back"
    checkAndroid6Permissions={true}
    fadeIn={true}
    notAuthorizedView={
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 16,
              "textAlign": "center",
            }
          }
        >
          Camera not authorized
        </Text>
      </View>
    }
    onRead={[Function]}
    pendingAuthorizationView={
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "fontSize": 16,
              "textAlign": "center",
            }
          }
        >
          ...
        </Text>
      </View>
    }
    permissionDialogMessage="Need camera permission"
    permissionDialogTitle="Info"
    reactivate={false}
    reactivateTimeout={0}
    showMarker={true}
    vibrate={true}
  />
  <ModalWebView
    handleButton={[Function]}
    modalVisible={false}
    openLink={[Function]}
    url=""
  />
</View>
`;
